packages/control/lib/metrics.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const RuntimeApiClient = require('./runtime-api-client')
+
+async function streamRuntimeMetricsCommand () {
+  const client = new RuntimeApiClient()
+  const runtime = await client.getMatchingRuntime([])
+  const metricsStream = client.getRuntimeLiveMetricsStream(runtime.pid)
```

Comment:
Unfortunately no, we should be should use the other API, `/metrics`.

Note this was just added to `control` in https://github.com/platformatic/platformatic/pull/3715.

Reply:
Rebased and updated ✅